package.json

```diff
@@ -1,20 +1,19 @@
 {
   "name": "exosphere-sdk",
-  "version": "0.22.1",
+  "version": "0.22.2-test.1",
```

Comment:
I'll revert this after this has been reviewed. You can checkout this branch and test out the script that fetches the go binary for yourself.